packages/hs-react-ui/src/components/InteractionFeedback/InteractionFeedback.tsx

```diff
@@ -0,0 +1,97 @@
+import React, { useCallback, useState } from 'react';
+import { animated, useTransition } from 'react-spring';
+import styled, { StyledComponentBase } from 'styled-components';
+import shortid from 'shortid';
+import colors from '../../enums/colors';
+import useResizeObserver from 'use-resize-observer/polyfilled';
+
+// TODO Reduce amount of any/ts-ignore done here
```

Comment:
I don't see any - is this TODO still relavent?

Reply:
Doesn't seem so, removed.